tests/Controller/SearchControllerApiTest.php

```diff
@@ -77,7 +77,7 @@ public function it_shows_product_list_page_from_WEB_GB_channel_filtered_by_taxon
 
         $this->assertProductsCodesInResponse(
             $this->client->getResponse(),
-            ['LOGAN_HAT_CODE', 'LOGAN_T_SHIRT_CODE', 'LOGAN_MUG_CODE']
+            ['LOGAN_MUG_CODE', 'LOGAN_HAT_CODE', 'LOGAN_T_SHIRT_CODE']
```

Comment:
What happened to sorting by positions?

Reply:
Hmm, I compacted taxons into one single sub-document (remember that we ended up having taxons and productTaxons that where basically the same), need to check the elastic query that it applies the sorting.

Reply:
@pamil hmm, in the SearchController you have added the sorting code - it checks if sort param is not defined, then you check that either the taxonCode or taxonSlug sort params are not null. So it results in no sorting as far as I understand.